packages/testcontainers/src/wait-strategies/http-wait-strategy.test.ts

```diff
@@ -1,9 +1,33 @@
 import { GenericContainer } from "../generic-container/generic-container";
 import { Wait } from "./wait";
 import { checkContainerIsHealthy, checkContainerIsHealthyTls } from "../utils/test-helper";
+import { getContainerRuntimeClient } from "../container-runtime";
+import { IntervalRetry } from "../common";
 
 jest.setTimeout(180_000);
 
+async function stopStartingContainer(container: GenericContainer, name: string) {
```

Comment:
I was thinking to move this function into the describe block as well, but actually it's generic enough to be moved to `test-helper.ts`